### ISSUE TITLE
Bundle creation not detected Net472 Package

### ISSUE THREAD

**Issue by Torchok19081986** [oleg-shilo/wixsharp#948]
Hallo i have trouble to add NetFx472 Package to bundle.
````csharp
  var bootstrapper =
              new Bundle("MyProduct",
                  new PackageGroupRef("NetFx472Redist"), //Or NetFx472Web either
                  new MsiPackage(productMsi)
                  {
                      Id = "PackageId",
                      DisplayInternalUI = true, 
                      Visible = true,
                  })
````
doesnt work. It works until i set  greater than NetFx461.
How do i set NetFx472 or greater to my bundle ?
i use visual studio 2019 and wix toolset in V3.11 with wixsharp 1.15.
Many Thanks.

**Comment by snechaev** [comment]
WIX 3.11 (more precisely, theirs NetFx472  extension) does not support net4.7+ (https://github.com/wixtoolset/issues/issues/5575). To make it works you can try the newest builds of WIX (but they are still in nightly/preview state) or try to implement support by yourself as described in the WIX issue referenced above.

The short description of my solution for net4.8
* grab corresponding wxs from wix repository or from the issue (NetFx48.wxs in my case) and include it in your project
* include grabbed wxs in your bundle with `bootstrapper.WxsFiles.Add("NetFx48.wxs".PathGetFullPath());`. This should be done before build.
* add `new PackageGroupRef("NetFx48Redist")` inside your bundle (you already done this).

**Comment by Torchok19081986** [comment]
Morning, @snechaev . Many thanks for help. I tried your solution. At creation of bundle , compiler want netfx472.wixobj (for my bundle) and cant find it. I add wxs file to my project .  Do i have to set wxs in build action as ressource or something else?

**Comment by snechaev** [comment]
Here is a minimal bootstrapper code, which compiles without issues for me with NetFx472
```
        var bootstrapper =
            new Bundle(productName,
                new PackageGroupRef("NetFx472Redist"),
                msiPackage
            );

        bootstrapper.Version = project.Version;
        bootstrapper.OutFileName = "setup-test";
        bootstrapper.WxsFiles.Add("NetFx472.wxs".PathGetFullPath());
        bootstrapper.Include(WixExtension.Util);
        return bootstrapper.Build();
```
Will this compile in you project?

**Comment by Torchok19081986** [comment]
Morning, @snechaev. Thanks for answer again. I tried your test solution. I have to additionally add NetFx extension to bootstrapper.
Without this , doesn't work, somehow and wix toolset compiler search or try to create <bootstrappername>. wixobj . Dont know why.

**Comment by snechaev** [comment]
Yes, I missed NetFx. I fixed my comment now to help anyone who may find this issue in a future.

So, after all you was able to build your project, right?

**Comment by Torchok19081986** [comment]
Yes. Now it builds successfully , and at end of compiletask i got my bundle with 4.7.2 NetFramework and tested it at VM also ok. Am i right , same task ist for Net Core or Net 5.0 ?

**Comment by snechaev** [comment]
Great!
As for net core/net5 I can't say for sure, but as I understood, the recommended way for distribution is self-contained apps, which do not need any prerequisites to be installed. 

Net core / net 5 still allows to build framework-dependent application, and such apps will require to install corresponding runtime.
But I am unsure that wix/NetFx supports such a runtimes. Higly likely it does not support. Thus, if you need to install net core/net5 runtime - you will need to perform all checks and install operations manually without any help from wix.

**Comment by oleg-shilo** [comment]
A very accurate assessment Sergey
